### PR TITLE
Merge SaveAndReturn branch

### PR DESCRIPTION
**Pull request by SandipKhedekar**

### CODE REVIEW

File: app/views/PrintLink.scala.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(key: String, id: Option[String] = None)(implicit messages: Messages)
+
+<a class="govuk-link" @if(id.isDefined){id="@id"} href="#" onclick="window.print();return false;">@messages(key)</a>
```
Comment (bharti5verma) [inline app/views/PrintLink.scala.html:21]: End of line missing

File: conf/messages.cy [hunk 2]
```diff
@@ -479,11 +468,87 @@ upliftFailed.details = cy: Alternatively, give us details from one or two of the
...
+
+confirmEditAnswers.title = cy: Do you want to edit your submission answers?
+confirmEditAnswers.heading = cy: Are you sure you want to edit your calculation answers?
+confirmEditAnswers.subHeading = cy: You will be sent to the task list in the calculation section. Any answers in the submission section you have already given will be deleted.
+confirmEditAnswers.error.required = cy: Select yes if you want to edit a previous submission
+
+confirmRestartAnswers.title = cy: Do you want to restart your calculation?
+confirmRestartAnswers.heading = cy: Do you want to restart your calculation?
+confirmRestartAnswers.message = cy: You will be sent to the beginning of the service. You will lose all progress that you have made so far.
+confirmRestartAnswers.error.required = cy: Select yes if you want to restart your calculation.
```
Comment (bharti5verma) [inline conf/messages.cy:554]: End of line missing

File: conf/messages.en [hunk 3]
```diff
@@ -487,3 +476,79 @@ incompleteData.title = You did not complete a change you were making
...
+
+confirmEditAnswers.title = Do you want to edit your submission answers?
+confirmEditAnswers.heading = Are you sure you want to edit your calculation answers?
+confirmEditAnswers.subHeading = You will be sent to the task list in the calculation section. Any answers in the submission section you have already given will be deleted.
+confirmEditAnswers.error.required = Select yes if you want to edit a previous submission
+
+confirmRestartAnswers.title = Do you want to restart your calculation?
+confirmRestartAnswers.heading = Do you want to restart your calculation?
+confirmRestartAnswers.message = You will be sent to the beginning of the service. You will lose all progress that you have made so far.
+confirmRestartAnswers.error.required = Select yes if you want to restart your calculation.
```
Comment (bharti5verma) [inline conf/messages.en:554]: End of line missing

**Review by ElliotAndersonHMRC — APPROVED** [review]
Looks good

**Review by bharti5verma — APPROVED** [review]

File: conf/messages.cy (hunk 2 above)
Comment (SandipKhedekar) [reply conf/messages.cy]: Done

File: conf/messages.en (hunk 3 above)
Comment (SandipKhedekar) [reply conf/messages.en]: Done

**Pull request merged**